.github/workflows/build-and-test.yml

```diff
@@ -125,3 +125,207 @@ jobs:
         with:
           name: collector-binaries
           path: ./bin
+  loadtestpre:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.splitloadtest.outputs.matrix }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.15
+      - name: Setup env
+        run: |
+          echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
+          echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
+      - name: Restore module cache
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-go-modules
+        with:
+          path: /home/runner/go/pkg/mod
+          key: go-pkg-mod-${{ runner.os }}-${{ hashFiles('./go.mod') }}
+      - name: split loadtest jobs
+        id: splitloadtest
+        run: |
+          TESTS="$(make -s testbed-list-loadtest | xargs echo|sed 's/ /|/g')"
+          TESTS=(${TESTS//|/ })
+          MATRIX="{\"include\":["
+          curr=""
+          for i in "${!TESTS[@]}"; do
+            if (( i > 0 && i % 2 == 0 )); then
+              curr+="|${TESTS[$i]}"
+            else
+              if [ -n "$curr" ] && (( i>1 )); then
+                MATRIX+=",{\"test\":\"$curr\"}"
+              elif [ -n "$curr" ]; then
+                MATRIX+="{\"test\":\"$curr\"}"
+              fi
+              curr="${TESTS[$i]}"
+            fi
+          done
+          MATRIX+="]}"
+          echo "::set-output name=matrix::$MATRIX"
+  loadtest:
+    runs-on: ubuntu-latest
+    needs: [setup-environment, loadtestpre]
```

Comment:
This should need binaries as well correct?

Reply:
We thought that too but found out it doesn't actually use any of the binaries so we moved it out. It seems to be running test suites instead

Reply:
The test suite starts one of the binaries so it needs the binary

Reply:
oh weird...I wonder why its passing when we're not including them. I will add it again. Thanks for the quick reviews!

Reply:
It is re-building, in the makefile we depend on that step so it will just rebuild the binary :)